lib/pronto/commentator.rb

```diff
@@ -44,7 +44,7 @@ def repo_path
     def config
       conf_file = File.join CONFIG_DIR, 'config.yml'
 
-      @config ||= YAML.load(File.read(conf_file))
+      @commentator_config ||= YAML.load(File.read(conf_file))
```

Comment:
Why this change? (It's generally idiomatic to name a "memoized" ivar the same thing as its method...)